src/context/MarriageOptionsContext.tsx

```diff
@@ -0,0 +1,28 @@
+import { createContext, ReactNode, useEffect, useState } from "react";
+import { HuwelijksplannerState, initialState } from "../data/huwelijksplanner-state";
+
+const myWindow = typeof window !== "undefined" ? window : undefined;
+
+const getSavedMarriageOptions = () => {
+  const savedOptions = myWindow?.sessionStorage.getItem("marriageOptions");
```

Comment:
Do we need `window.sessionStorage.getItem(...)`? I wonder why we wouldn't be able to just use `sessionStorage.getItem(...)` instead?

Reply:
I think after a small refactoring I did later this can indeed be removed for this method